sources/editor/Stride.Core.Assets.Editor.Avalonia/NewOrOpenSessionTemplateCollectionViewModel.cs

```diff
@@ -0,0 +1,21 @@
+using Stride.Core.Presentation.Commands;
```

Comment:
Note: view models should be as much as possible outside Avalonia projects. The idea being that they could be reused by another UI library if it were to change in the future (or if we decide to keep WPF around and share mist of the view models between the two).

It implies that view models cannot use or reference any UI-specific types. Even without reuse, it helps keeping the view and model logic separated.

Reply:
I agree, I moved VMs here temperarily for just convenience 😅